source/setup/components/ArchiveUnlockPage.js

```diff
@@ -45,9 +21,9 @@ class ArchiveUnlockPage extends Component {
     };
 
     componentDidMount() {
-        setTimeout(() => {
+        if (this._passwordInput) {
```

Comment:
This was originally a timeout due to threading and the fact that the input wasn't actually ready when `componentDidMount` was called. It never focused while executed synchronously. Did you find any issues with this?

Reply:
@perry-mitchell something must have changed in react... because now it just works without the setTimeout. however if you think this is not safe we can restore the setTimeout?